internal/upgrader/upgrader.go

```diff
@@ -128,6 +129,9 @@ func performUpgrade(api CFClient, upgradableInstances []ccapi.ServiceInstance, p
 	})
 
 	log.FinalTotals()
+	if !log.HasUpgradeSucceeded() {
+		return errors.New("there were failures upgrading one or more instances. Review the logs for more information")
```

Comment:
```suggestion
		return errors.New("there were failures upgrading one or more instances. Previously, this resulted in a 0 exit code anyway but a more sensible behaviour was introduced in https://github.com/cloudfoundry/upgrade-all-services-cli-plugin/pull/117. Please adapt any scripts accordingly. Review the logs for more information")
```

Reply:
Discussed and agreed not to change the message as it will be noted in release notes. We would also like to avoid having to think when to clean up the messaging.